cli/on-boarding/terraform.md

```diff
@@ -5,73 +5,118 @@ description: Learn how to onboard Terramate to any existing Terraform project wi
 
 # Get started with Terramate and Terraform
 
-This page will teach you how to onboard Terramate to an existing Terraform project.
+In this guide, You will learn how to add Terramate to an existing Terraform project without changing any configuration.
 
-Terramate can be added to any existing Terraform project with a single command
-and without changing any existing configuration, making the onboarding **seamless**
-and **fast**.
+## 1: Install Terramate
 
-::: tip
-If instead, you'd like to learn how to get started with Terramate from scratch, take a look at our
-[Getting Started Guide](../getting-started/index.md).
-:::
+ Install the [Terramate cli](../installation.md).
```

Comment:
@AnnuCode can we at least add `brew install terramate` here and in case a user needs other installation methods, can we refer to https://terramate.io/docs/cli/installation#install-the-cli

Reply:
yes, that would be good. Thanks!